react-design/react-design/src/Selections.tsx

```diff
@@ -0,0 +1,16 @@
+type Props = {
+  onSelection: (item: string) => void;
```

Comment:
If you passed the index of the button instead of the string it would make the client code simpler. Also, right now it won't work correctly if two items have exactly the same text.